src/reducers/ui.js

```diff
@@ -360,12 +361,28 @@ export const sGetDimensionIdsFromLayout = (state) =>
         []
     )
 
+export const sGetUiDimensionIdsForLayoutAxis = (state, axisId) =>
```

Comment:
rename to fit in to the other selectors?
```suggestion
export const sGetUiDimensionIdsByAxisId = (state, axisId) =>
```